app/Http/Controllers/WordController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Word;
+use App\WordAnswer;
+use App\Category;
+
+class WordController extends Controller
+{
+    public function index(Category $category)
+    {
+        $word = new Word;
+        $words = $word->findWords($category);
+        $answer = new WordAnswer;
+        $answers = $answer->correctAnswers();
+        return view('admin.wordList')->with([
+            'words' => $words,
+            'answers' => $answers,
+            'category' => $category
+        ]);
+    }
+
+    public function add(Category $category)
+    {
+        $category_id = $category->id;
+        return view('admin.addWord')->with('category_id', $category_id);
+    }
+
+    public function create(Request $request)
+    {
+        //Creating the word.
+        $word = new Word;
+        $word->create($request);
+
+        //Creating the answers for the word.
+        $word = new Word;
+        $word_id = $word->getLatestId();
+        for($i=1; $i<=5; $i++) {
+            $choicex = "choice". $i;
+            $correct = $request->check === $choicex ? 1 : 0;
+            $word_answer = new WordAnswer;
```

Comment:
please use the relationships of Word and WordAnswer, you can create a code more clean and simple way.